scipy/ndimage/tests/test_measurements.py

```diff
@@ -771,6 +771,7 @@ def test_minimum03(xp):
         assert_almost_equal(output, xp.asarray(2.0), check_0d=False)
 
 
+@skip_xp_backends('dask.array', reason="no argsort in Dask")
 def test_minimum04(xp):
```

Comment:
It is worth noting that we do implement [`minimum` in dask-image]( https://github.com/dask/dask-image/blob/77a29ba78db608c5fe21f9f51745ec4bd01d5681/dask_image/ndmeasure/__init__.py#L619 ) and do so without `argsort`

Would be interested to know whether there might be a way to benefit from the existing dask-image work here